### PR TITLE
Wish bump to 1.3.1

### PR DESCRIPTION
**Pull request by MauriceX24**
<details>
<summary>Changes</summary>
<br>
- Fixed product information
<br>
- Added TypeScript
</details>

### CODE REVIEW

File: Wish/dist/metadata.json [hunk 1]
```diff
@@ -13,27 +13,21 @@
   "logo": "https://i.imgur.com/XsZ2IOn.jpg",
   "thumbnail": "https://i.imgur.com/R7AkWiP.png",
   "color": "#48c0eb",
-  "tags": [
-    "Shopping",
-    "Shop"
-  ],
+  "tags": ["Shopping", "Shop"],
   "category": "other",
```
Comment (Bas950) [inline Wish/dist/metadata.json:17]: remove the cap letters

File: Wish/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        },
+		{
+            "id": "language",
+            "title": "Language",
+            "icon": "fas fa-language",
+            "value": 0,
+            "values": ["English", "German"]
+        }
+	]
+}
```
Comment (Bas950) [inline Wish/tsconfig.json:33]: https://docs.premid.app/dev/presence#filling-in-the-tsconfigjson-file

**Comment by MauriceX24** [comment]
I already see the deepscan error and maybe you can explain me tomorrow? @Bas950

**Comment by doomlerd** [comment]
Update your compiler to the latest version.

**Comment by MauriceX24** [comment]
Can you close this please :o
I will work later again .. I have to learn how I have to work with ts xd

**Comment by doomlerd** [comment]
Well you're really close to finishing but as you wish.

**Pull request closed without merging**